micromasters/settings.py

```diff
@@ -498,3 +498,26 @@ def get_var(name, default):
 # Open Exchange Rates
 OPEN_EXCHANGE_RATES_URL = get_var("OPEN_EXCHANGE_RATES_URL", None)
 OPEN_EXCHANGE_RATES_APP_ID = get_var("OPEN_EXCHANGE_RATES_APP_ID", "")
+
+# django debug toolbar only in debug mode
+if DEBUG:
+    INSTALLED_APPS += ('debug_toolbar', )
+    # it needs to be enabled before other middlewares
+    MIDDLEWARE_CLASSES = (
+        'debug_toolbar.middleware.DebugToolbarMiddleware',
+    ) + MIDDLEWARE_CLASSES
+
+    def show_toolbar(request):
+        """
+        Custom function needed because of bug in wagtail.
+        Theoretically this bug has been fixed in django 1.10 and wagtail 1.6.3
```

Comment:
I think we need to check if this is still an issue after we upgrade to django 1.10